SKILL.tsv

```diff
@@ -5354,46 +5354,46 @@ SKILL_20151016_005353	$Causes instant kill on small-type monsters by 0.5% chance
 SKILL_20151022_005366	$Sept Etoiles
-SKILL_20151022_005367	
+SKILL_20151022_005367	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}Stab an enemy with consecutive strikes.
 SKILL_20151022_005368	$Attaque Coquille
-SKILL_20151022_005369	
+SKILL_20151022_005369	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}Attack an enemy with the guard of your sword. The enemy's defence is ignored when attacked again with [Pierce] attacks.
 SKILL_20151022_005370	$Esquive Toucher	
-SKILL_20151022_005371	
-SKILL_20151022_005372	
+SKILL_20151022_005371	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}Move swiftly to avoid enemy attacks and deliver complex consecutive attacks.
+SKILL_20151022_005372	$Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Evasion Rate: +#{CaptionRatio2}#%
 SKILL_20151022_005373	$Preparation
-SKILL_20151022_005374	
-SKILL_20151022_005375	
+SKILL_20151022_005374	$Twirl the tip of your sword to parry an enemy attack and prepare for the next attack. The next [Pierce] attack you make will inflict twice as much damage.
+SKILL_20151022_005375	$Maximum Duration: #{CaptionRatio2}# seconds{nl}Block Rate +#{CaptionRatio}#%{nl}Duration: 15 seconds
 SKILL_20151022_005376	$Flanconnade
-SKILL_20151022_005377	
+SKILL_20151022_005377	${#DD5500}{ol}[Physical] - [Slash]{/}{/}Perform a complex upward attack with your rapier. This attack temporarily increases your block rate.
 SKILL_20151022_005378	
 SKILL_20151022_005379	
 SKILL_20151022_005380	
 SKILL_20151022_005381	
-SKILL_20151022_005382	
-SKILL_20151022_005383	
-SKILL_20151022_005384	
-SKILL_20151022_005385	
+SKILL_20151022_005382	${#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Cast an arrow of ice that inflicts damage and has a chance of freezing the enemy.
+SKILL_20151022_005383	${#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Make ice spikes sprout out in a line, inflicting damage and freezing enemies by chance.
+SKILL_20151022_005384	${#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Ride on an enormous snowball and crush any enemies in your way.
+SKILL_20151022_005385	$Attack: 400% +#{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Casting Time: #{CaptionTime}#seconds{nl}AoE Attack Ratio: #{SkillSR}#
 SKILL_20151022_005386	
 SKILL_20151022_005387	
 SKILL_20151022_005388	
 SKILL_20151022_005389	
 SKILL_20151022_005390	
 SKILL_20151022_005391	
-SKILL_20151022_005392	
+SKILL_20151022_005392	$Carve a sculpture of Ausrine, the Goddess of the Morning Star. Those in your party who circle around the sculpture counter-clockwise are granted divine protection and become immune to damage temporarily.
 SKILL_20151022_005393	
-SKILL_20151022_005394	
-SKILL_20151022_005395	
-SKILL_20151022_005396	
+SKILL_20151022_005394	$You and your party members are temporarily granted an increase in fire, ice, lightning, poison, and earth resistances, and have a chance of nullifying attacks from those properties.
+SKILL_20151022_005395	$Increased Resistance: +#{CaptionRatio}#{nl}Nullification Chance: #{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds
```

Comment:
$Increased Resistance: +#{CaptionRatio}#{nl}Nullification Chance: #{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds

to

$Property Resistance: +#{CaptionRatio}#{nl}Nullification Chance: #{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds